### ISSUE TITLE
get build id when building new job

### ISSUE THREAD

**Issue by orevron** [jenkins-zh/jenkins-cli#430]
currently i can't find a way to get the build id of a recently triggered build.
let's say i run the following command:
* jcli job build my-job -b

the following command i'd like to use is:
* jcli job log my-job 

to get the recently triggered job, i acknowledged that if i run this command without build id i get the latest build.
but what if other employee in my company just triggered the job right after me?

bottom line i'd love to have the build id for a job i just triggered as the response to the build command.

thanks!

**Comment by LinuxSuRen** [comment]
Thanks to file this issue. You're right. There's a direct way to the build ID after your triggered a job. Because of Jenkins puts the build request into a queue first, then handle it by another worker.
But I'd love to resolve this problem. A lot of users wish to give the build ID.
